tests/Sdk/Unit/Trace/SpanProcessor/BatchSpanProcessorTest.php

```diff
@@ -144,15 +145,13 @@ public function forceFlushExportsAllEndedSpans()
         $clock->method('timestamp')->will($this->returnValue(($exportDelay - 1)));
 
         $exporter = self::createMock(Exporter::class);
-        /** @var \OpenTelemetry\Sdk\Trace\Exporter $exporter */
```

Comment:
Interestingly, the correct course of action here was to actually remove type information since it was incorrect, removing the PHPUnit mock type from the union.g